lambda-assets/extensions/cf-default-dir-index/index.ts

```diff
@@ -1,18 +1,17 @@
-exports.handler = (event, context, callback) => {
-    
-  // Extract the request from the CloudFront event that is sent to Lambda@Edge 
+export async function handler(event, context, callback) {
```

Comment:
```suggestion
export async function handler(event: any) {
```

I think this would be enough.

Reply:
And the last line would be

```ts
return request;
```

Reply:
LGTM